cmd/test.go

```diff
@@ -16,16 +16,24 @@ import (
 
 	"github.com/pelletier/go-toml/v2"
 	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
```

Comment:
We can bump the Go version in `go.mod` to 1.22 minimum and avoid importing an `x/exp` package, those utilities have been stabilized.

Reply:
Shall we do it in this PR or separate one? I'd advocate for separate

Reply:
Agreed!

Reply:
They are [still not included in go 1.22.3](https://github.com/golang/go/tree/go1.22.3/src/maps)

I see the functions I use are [in master](https://github.com/golang/go/tree/master/src/maps) though (under `iter.go`), so hopefully in go 1.23 we will have them

Reply:
Woof, I thought they were in already :-/